### PR TITLE
Feature/agenda

### PR DESCRIPTION
**Pull request by riyaelizabeth**
- changes  done

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+//         console.log("error in connecting to db")
+//     })
 const viewCurrentBookingsQuery = require('./cron/show_time')
     //const sendNotification = require('./cron/show_time')
 const dotenv = require('dotenv');
 var moment = require('moment');
+const mongoConnectionString = 'mongodb://127.0.0.1/agenda';
+
+//const agenda = new Agenda({ db: { address: mongoConnectionString, collection: 'Bookings' } });
 moment().format();
```
Comment (experionsachin) [inline server.js:17]: What is this?

File: app/modules/jobs/agenda.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    expTickets = await tickets.findAll({
+        showTime: Sequelize.literal('showTime + 10,800,000')
+    }, {
+        where: {
+            showTime: {
+                [Op.lt]: moment.utc().valueOf()
+            }
+        }
+    })
+    console.log(expTickets)
```
Comment (experionsachin) [inline app/modules/jobs/agenda.js:17]: remove consoles

File: app/modules/jobs/agenda.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+//const { Tickets: tickets } = require('../../../models')
+const { tickets } = require('../../../../models/index');
+
+const Agenda = require('agenda')
+const mongoConnectionString = 'mongodb://127.0.0.1/agenda';
+agenda = new Agenda({ db: { address: mongoConnectionString } });
+agenda.define('findExpiredTickets', async job => {
+    expTickets = await tickets.findAll({
```
Comment (experionsachin) [inline app/modules/jobs/agenda.js:8]: const or let ??

**Review by experionsachin — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (riyaElizabeth) [reply server.js]: As per the docs, giving connection and creating object .

**Pull request closed without merging**